packages/vow/src/watch-utils.js

```diff
@@ -0,0 +1,106 @@
+// @ts-check
+
+import { M } from '@endo/patterns';
+
+const VowShape = M.tagged(
+  'Vow',
+  harden({
+    vowV0: M.remotable('VowV0'),
+  }),
+);
+
+/**
+ * @param {import('@agoric/base-zone').Zone} zone
+ * @param {import('./watch.js').Watch} watch
+ * @param {() => import('./types.js').VowKit<any>} makeVowKit
+ */
+export const prepareWatchUtils = (zone, watch, makeVowKit) => {
+  const detached = zone.detached();
+  const makeWatchUtilsKit = zone.exoClassKit(
+    'WatchUtils',
+    {
+      utils: M.interface('Utils', {
+        all: M.call(M.arrayOf(VowShape)).returns(M.any()),
```

Comment:
`all` should accept and return arrays of any values (doesn't matter what they are), whether Vow, Promise, or any other passable.
```suggestion
        all: M.call(M.arrayOf(M.any())).returns(M.arrayOf(M.any())),
```